README.md

```diff
@@ -1,25 +1,25 @@
-These are a series of javascript exercises intended to be used alongside the curriculum at 'The Odin Project'. They start simple and easy, but get more complex and involved as you progress through them.
+These are a series of JavaScript exercises intended to be used alongside the curriculum at 'The Odin Project'. They start simple and easy but get more complex and involved as you progress through them.
 
-There will eventually be a suggested order of completion, but at this time since we are still in the process of creating more exercises the order is subject to change and has not yet been specified. However, there are a few exercises which make a good "starting point". Feel free to at least start with these:
+There will eventually be a suggested order of completion, but at this time since we are still in the process of creating more exercises the order is subject to change and has not yet been specified. However, there are a few exercises that make a good "starting point". Feel free to at least start with these:
 
 1. Hello World
-1. Repeat String
-1. Reverse String
+2. Repeat String
+3. Reverse String
 
-## HOW TO USE THESE EXERCISES
+## How to Use These Exercises
 Before you start you should have a few things installed on your machine:
-1. NPM.  To check if you have NPM, type `npm --version` in a terminal. If you get back `Command 'npm' not found, but can be installed with:`, do NOT follow the instructions in the terminal to install with `apt-get` (this causes permission issues). Instead, install NPM/Node with NVM by following the instructions [here](https://github.com/TheOdinProject/curriculum/blob/master/foundations/installations/installing_node.md).
-3. Jasmine.  Jasmine is a testing framework for Javascript.  Type `jasmine -v` to check for it.  If you need to install it, type `npm install -g jasmine` to do so.
-4. Clone this repo and get started.
+1. NPM.  To check if you have NPM installed, type `npm --version` in your terminal. If you get back `Command 'npm' not found, but can be installed with:`, do NOT follow the instructions in the terminal to install with `apt-get`. (This causes permission issues.) Instead, install Node with NVM by following the instructions [here](https://github.com/TheOdinProject/curriculum/blob/master/foundations/installations/installing_node.md).
+2. Jasmine.  Jasmine is a testing framework for JavaScript. Type `jasmine -v` to check for it. If you need to install it, type `npm install -g jasmine` to do so.
+3. A copy of this repository. Copies of repositories on your machine are called clones. If you need help cloning, you can learn how [here](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/duplicating-a-repository)
```

Comment:
At this point, the students should already be familiar with cloning due to other prior lessons.

Reply:
I am going through Foundations, and there's almost no practice with cloning. There was likely brief exposure to it, but not enough to be able to copy the repository without first searching on Google.

Reply:
We have multiple lessons where one indeed does clone repositories- that said, after some thought, I don't see the need to remove this so we can keep it, given that people can potentially contribute at any stage in their learning 😄